ecs-deploy

```diff
@@ -771,20 +782,25 @@ if [ "$BASH_SOURCE" == "$0" ]; then
         exit 0
     fi
 
-    # Determine image name
-    parseImageName
-    echo "Using image name: $useImage"
+    if [[ "$USE_EXISTING_TASK_DEFINITION_ARN" != false ]]; then
+        NEW_TASKDEF="$USE_EXISTING_TASK_DEFINITION_ARN"
```

Comment:
I feel like `TASKDEF_TO_USE` or similar might capture what this variable is really for a bit better, but I think it's fine as is for our fork / not sure what the ecs-deploy maintainers' take on this would be

Reply:
yeah i didn't want to rename a bunch of existing stuff based on this one use case so i'm gonna leave it as is for now